lesson_01/task_02/index.js

```diff
@@ -0,0 +1,23 @@
+import fs from 'fs';
+import csv from 'csvtojson';
+
+const onError = err => {
+    console.log(err.message)
+};
+const onComplete = () => {
+    console.log('Task completed');
+};
+const onNext = json => {
+    console.log(json)
+};
+
+const rStream = fs.createReadStream('./assets/csv/nodejs-hw1-ex1.csv');
```

Comment:
it's better to use `path` library  to avoid problems with other OSs

Reply:
I think, done